src/main/resources/assets/nochatreports/lang/en_us.json

```
@@ -10,5 +10,40 @@
 	"gui.nochatreports.unsafe_server.header": "Caution: Unsafe Server",
 	"gui.nochatreports.unsafe_server.contents": "This server requires all chat messages to be signed, which means they can be reported to Mojang, which may result in a ban from all multiplayer servers. This likely means that the server owner kept \"enforce-secure-profile\" enabled in the server.properties, which is the default since 1.19.1. You can choose to play anyway, but No Chat Reports will not be able to protect you.",
 	"gui.nochatreports.unsafe_server.check": "Do not show again for this server",
-	"gui.nochatrepords.reload_config_tooltip": "Reload No Chat Reports config. Use carefully."
-}
+	"gui.nochatrepords.reload_config_tooltip": "Reload No Chat Reports config. Use carefully.",
+
+	"configuration.NoChatReports.config": "No Chat Reports",
+	"configuration.NoChatReports.category.visual": "Visual",
+	"configuration.NoChatReports.category.technical": "Technical",
+	"configuration.NoChatReports.category.whitelistedServers": "Whitelisted Servers",
+	"option.NoChatReports.demandOnServer": "Demand on server",
+	"option.NoChatReports.demandOnServer.tooltip": "Prevents the client from joining servers that don't\nhave No Chat Reports installed.",
+	"option.NoChatReports.enableDebugLog": "Enable debug log",
+	"option.NoChatReports.enableDebugLog.tooltip": "Enables logging of certain debug information.\nThis is generally useless for non-developers.",
+	"option.NoChatReports.convertToGameMessage": "Convert to game message",
+	"option.NoChatReports.convertToGameMessage.tooltip": "Makes the server convert all player messages into system\nmessages. System messages don't have any signatures\nattached and cannot be reported; on unmodified clients\nthey will display with gray bar to the left.",
+	"option.NoChatReports.showServerSafety":"Show server safety",
+	"option.NoChatReports.showServerSafety.tooltip":"Allows the client to display server safety status in\nbottom-right corner of chat screen. The icon is\nhoverable for additional information:",
```

```suggestion
	"option.NoChatReports.showServerSafety.tooltip":"Allows the client to display server safety status in\nbottom-right corner of chat screen. The icon is\nhoverable for additional information.",
```
Why does the string end with a colon?

---

I copied and pasted from the wiki and forgot to remove it